src/routes/lobby/LobbyView.vue

```diff
@@ -59,7 +59,9 @@
                   data-cy="edit-game-ranked-switch"
                   color="primary"
                   hide-details
+                  tabindex="0"
```

Comment:
Why the tabindex? You can already focus the element with the tab key in the regular order of the document flow, and actually the 'Ready' button above is the more primary action which should be focused first. I think we can skip this
```suggestion
```